Change Mo topic check

AbhayGoyal opened the pull request:
Changed the topic and timezone for Mo Akhshik

AbhayGoyal committed:
Change Mo topic check

canyon289 commented on _data/speakers.yml:
```diff
@@ -63,18 +63,18 @@
     Broadly interested in cosmology and probability too.
   presentation_type: Talk
-  time_zone:
+  time_zone: 9-15 GMT+2
```
Per oriols suggestion I would change this naming to what was said in slack

AbhayGoyal replied:
Yup. I guess we can change it in the sheet as well?

AbhayGoyal replied:
$\sim$3 pat in the summary looks a bit unnecessary. Should I remove it?

canyon289 replied:
Its fine in the sheet

canyon289 commented on _data/speakers.yml:
```diff
@@ -63,18 +63,18 @@
-  topic:
-  proposal_summary:
+  topic: A Novel Bayesian Model to Fit Spectrophotometric Data of Hubble and Spitzer Space Telescopes
+  proposal_summary: 
```
Can we add the summary as well?

AbhayGoyal replied:
Did change the summary. pushed it

AbhayGoyal committed:
Changed Mo topic summary

OriolAbril commented on _data/speakers.yml:
```diff
@@ -63,18 +63,18 @@
-  topic:
-  proposal_summary:
+  topic: A Novel Bayesian Model to Fit Spectrophotometric Data of Hubble and Spitzer Space Telescopes
+  proposal_summary: Understanding how the most massive galaxies rapidly formed and quenched when Universe was only $\sim$3 billion years old is one of the major challenges of extragalactic astronomy. In this talk, I will discuss how to improve our understanding of massive galaxy formation by combining the spectro-photometric observations of the Hubble and Spitzer Space Telescopes for strong gravi ...
```
can you remove the latex things? Unluckily they won't work on the website and will look weird:

* `$\sim$3` -> `~`
* `\texttt{pymc3}` -> PyMC3

AbhayGoyal replied:
Changed.

canyon289 commented:
@AbhayGoyal I have to jump off again but basically here's what will happen

Once this is good Oriol will merge this pull request into the main code branch. I dont actually think its going to show up on the website yet because we havent coded it, but youve basically gotten it ready so once were ready to make the changes on the website all the text will already be available!

As youve got time itd be great if you can add however many more. What I would suggest for the next few is making a pull request early so people know youre working, and in these comments mention which speakers youll be working on so we can coordinate without running into each other

AbhayGoyal committed:
Done till Micheal Johns

AbhayGoyal committed:
Done all

AbhayGoyal committed:
Made change asked Oriol

canyon289 commented:
This is great! I'll let @OriolAbril review since hes working on the grouping but I have no specific suggestion. Thrilled youre making so much progress on this pull request :)

OriolAbril commented on _data/speakers.yml:
```diff
@@ -24,7 +24,7 @@
   github: vpriesem
   web: http://www.viola-priesemann.de/
-  image: Viola.jpg
+ image: Viola.jpg
```
```suggestion
  image: Viola.jpg
```

OriolAbril commented on _data/speakers.yml:
```diff
@@ -63,19 +63,20 @@
-  topic:
-  proposal_summary:
+  topic: A Novel Bayesian Model to Fit Spectrophotometric Data of Hubble and Spitzer Space Telescopes
+  proposal_summary: Understanding how the most massive galaxies rapidly formed and quenched when Universe was only ~$3 billion years old is one of the major challenges of extragalactic astronomy. In this talk, I will discuss how to improve our understanding of massive galaxy formation by combining the spectro-photometric observations of the Hubble and Spitzer Space Telescopes for strong gravitati ...
```
```suggestion
  proposal_summary: Understanding how the most massive galaxies rapidly formed and quenched when Universe was only ~3 billion years old is one of the major challenges of extragalactic astronomy. In this talk, I will discuss how to improve our understanding of massive galaxy formation by combining the spectro-photometric observations of the Hubble and Spitzer Space Telescopes for strong gravitationally lensed galaxies. In particular, a multi-level regression model is built that can fit all multi-wavelength data for a range of instruments within a hierarchical Bayesian framework to constrain the properties of the stellar populations. The details of how this model is implemented using PyMC3, as well as the estimates of the posteriors of all parameters of interest and nuisance parameters will be highlighted.
```

OriolAbril commented on _data/speakers.yml:
```diff
@@ -63,19 +63,20 @@
     Broadly interested in cosmology and probability too.
   presentation_type: Talk
-  time_zone:
+  time_zone: European
```
```suggestion
  time_zone: Africa/Asia/Europe
```

OriolAbril commented on _data/speakers.yml:
```diff
@@ -136,6 +139,7 @@
   presentation_type: Let's Build a Model
   time_zone:
   track: Advanced
+  keywork: 
```
Everybody must have a keyword, if he takes part in 2 talks he should have the keyword of the "main" talk.

AbhayGoyal replied:
Done this, added vincent's keyword to Matthijs

OriolAbril commented on _data/speakers.yml:
```diff
@@ -248,17 +256,18 @@
-  time_zone:
+  time_zone: Asia/Africa/Europe
   track: Beginner
+  keywork: Laukes
```
The keyword needs to be one per talk, otherwise the grouping won't work. Here however, Laura has `Laukes` and below Michael Osthege has `Micege`. They are giving a talk between the two. In order to be able to group them, they should both have the same keyword, maybe `calibr8` or anything else, but it must be one per talk, not one per speaker. Sorry to be so nitpicky, but otherwise the grouping won't work

AbhayGoyal replied:
Did this

OriolAbril commented on _data/speakers.yml:
mlda multilevel sampler has 3 speakers, all should have the same keyword.

OriolAbril commented on _data/speakers.yml:
mlda multilevel sampler has 3 speakers, they should all have the same keyword.

AbhayGoyal replied:
Done this

OriolAbril commented on _data/speakers.yml:
```diff
@@ -347,17 +360,18 @@
-  time_zone:
+  time_zone: Americas
   track: Advanced
+  keywork: Osvtin
```
Sequential monte carlo talk has 2 speakers, both should have the same keyword.

AbhayGoyal replied:
Done

OriolAbril commented on _data/speakers.yml:
mlda multilevel sampler has 3 speakers, all should have the same keyword.

AbhayGoyal replied:
Done

OriolAbril commented on _data/speakers.yml:
```diff
@@ -428,17 +445,18 @@
-  time_zone:
+  time_zone: Asia/Africa/Europe
   track: Beginner
+  keywork: Micege
```
Michael should have the same keyword as Laura, the first and longer comment above.

AbhayGoyal replied:
Changed the keywork

OriolAbril commented on _data/speakers.yml:
```diff
@@ -471,17 +490,20 @@
-  time_zone:
+  time_zone: Americas
   track: Advanced
+  keywork: Pedrez
```
Sequential monte carlo talk has 2 speakers, both should have the same keyword.

AbhayGoyal replied:
Changed the keywork

OriolAbril reviewed (commented):
Thanks for all the work! This looks good.

I have added a couple suggestions to fix typos and very minor changes and then some more structural changes in order to get the grouping to work.

Grouping needs the following:
* All speakers need to have `timezone`, `track` and `keyword` filled, keynote speakers, people giving 2 talks, everyone
* the `keyword` field must be one per talk, otherwise we can't group speakers of the same talk together. I have commented on some of them to give some more clear and specific example, but there may be more talks given by multiple people.

canyon289 commented:
> Thanks for all the work! This looks good.
> 
> I have added a couple suggestions to fix typos and very minor changes and then some more structural changes in order to get the grouping to work.
> 
> Grouping needs the following:
> 
> * All speakers need to have `timezone`, `track` and `keyword` filled, keynote speakers, people giving 2 talks, everyone
> * the `keyword` field must be one per talk, otherwise we can't group speakers of the same talk together. I have commented on some of them to give some more clear and specific example, but there may be more talks given by multiple people.

@OriolAbril Not all speakers will have a timezone. Elizaveta for example didnt commit to a synchronous Q&A, which is totally acceptable, so she wont have one. 

There are also a couple missing of timezones for people that havent responded yet so there will be some blanks for now

OriolAbril commented:
We should then add a placeholder or give a name to not taking part in the live Q&A so they are all grouped together at the end or something like this

canyon289 commented:
Ok lets say `To be determined` for now, and then we fill in as they come

hectormz commented:
I think `keyword` got copied as `keywork` throughout the file

AbhayGoyal committed:
Update _data/speakers.yml

Co-authored-by: Oriol Abril-Pla <oriol.abril.pla@gmail.com>

AbhayGoyal committed:
Update _data/speakers.yml

Co-authored-by: Oriol Abril-Pla <oriol.abril.pla@gmail.com>

AbhayGoyal committed:
Update _data/speakers.yml

Co-authored-by: Oriol Abril-Pla <oriol.abril.pla@gmail.com>

AbhayGoyal commented:
I am still confused about Matthijs Brouns. He is giving 2 talks, one himself and the other is with Vincent, what would his keywork be then?

AbhayGoyal committed:
Updated

Changes made to all the speakers which are giving together. Their keywork is that of the first speaker.

hectormz commented on _data/speakers.yml:
```diff
@@ -63,19 +63,20 @@
-  time_zone:
+  time_zone: Africa/Asia/Europe
   track: Advanced
+  keywork: mohik
```
```suggestion
  keyword: mohik
```

AbhayGoyal replied:
Hey Hector, I had done that on purpose actually. Should I change them all?

hectormz commented on _data/speakers.yml:
```diff
@@ -96,8 +97,9 @@
-  time_zone:
+  time_zone: Africa/Asia/Europe
   track: Advanced
+  keywork: Alerra
```
```suggestion
  keyword: Alerra
```

hectormz commented on _data/speakers.yml:
```diff
@@ -108,14 +110,15 @@
-  time_zone:
+  time_zone: Americas
   track: Beginner
+  keywork: Aguelo
```
```suggestion
  keyword: Aguelo
```

hectormz commented on _data/speakers.yml:
```diff
@@ -136,6 +139,7 @@
   presentation_type: Let's Build a Model
   time_zone:
   track: Advanced
+  keywork: Vindam
```
```suggestion
  keyword: Vindam
```

hectormz commented on _data/speakers.yml:
```diff
@@ -146,16 +150,17 @@
-  time_zone:
+  time_zone: Asia/Africa/Europe 
   track: Advanced
+  keywork: Hesehr
```
```suggestion
  keyword: Hesehr
```

hectormz commented on _data/speakers.yml:
```diff
@@ -187,16 +192,17 @@
-  time_zone:
+  time_zone: Americas
   track: Beginner
+  keywork: Camlon
```
```suggestion
  keyword: Camlon
```

hectormz commented on _data/speakers.yml:
```suggestion
  keyword: Timwell
```

hectormz commented on _data/speakers.yml:
```diff
@@ -227,17 +234,18 @@
-  time_zone:
+  time_zone: Americas
   track: Beginner
+  keywork: Allney
```
```suggestion
  keyword: Allney
```

hectormz commented on _data/speakers.yml:
```diff
@@ -248,17 +256,18 @@
-  time_zone:
+  time_zone: Asia/Africa/Europe
   track: Beginner
+  keywork: Micege
```
```suggestion
  keyword: Micege
```

hectormz commented on _data/speakers.yml:
```diff
@@ -269,16 +278,17 @@
-  time_zone:
+  time_zone: Americas
   track: Beginner
+  keywork: Michns
```
```suggestion
  keyword: Michns
```

hectormz commented on _data/speakers.yml:
```diff
@@ -289,16 +299,17 @@
-  time_zone:
+  time_zone: Asia/Africa/Europe
   track: Beginner
+  keywork: Saymar
```
```suggestion
  keyword: Saymar
```

hectormz commented on _data/speakers.yml:
```diff
@@ -309,15 +320,16 @@
-  time_zone:
+  time_zone: Asia/Africa/Europe
   track: Beginner
+  keywork: Junlao
```
```suggestion
  keyword: Junlao
```

hectormz commented on _data/speakers.yml:
```suggestion
  keyword: Timwell
```

hectormz commented on _data/speakers.yml:
```diff
@@ -347,17 +360,18 @@
-  time_zone:
+  time_zone: Americas
   track: Advanced
+  keywork: Osvtin
```
```suggestion
  keyword: Osvtin
```

hectormz commented on _data/speakers.yml:
```diff
@@ -368,16 +382,17 @@
-  time_zone:
+  time_zone: Asia/Africa/Europe
   track: Advanced
+  keywork: Hesehr
```
```suggestion
  keyword: Hesehr
```

hectormz commented on _data/speakers.yml:
```suggestion
  keyword: Timwell
```

hectormz commented on _data/speakers.yml:
```diff
@@ -408,16 +424,17 @@
-  time_zone:
+  time_zone: Americas
   track: Advanced
+  keywork: Quayen
```
```suggestion
  keyword: Quayen
```

hectormz commented on _data/speakers.yml:
```diff
@@ -428,17 +445,18 @@
-  time_zone:
+  time_zone: Asia/Africa/Europe
   track: Beginner
+  keywork: Micege
```
```suggestion
  keyword: Micege
```

hectormz commented on _data/speakers.yml:
```diff
@@ -450,17 +468,18 @@
-  time_zone:
+  time_zone: Asia/Africa/Europe
   track: Beginner
+  keywork: Lucpaz
```
```suggestion
  keyword: Lucpaz
```

hectormz commented on _data/speakers.yml:
```diff
@@ -471,17 +490,20 @@
-  time_zone:
+  time_zone: Americas
   track: Advanced
+  keywork: Pedrez
```
```suggestion
  keyword: Pedrez
```

hectormz commented on _data/speakers.yml:
```diff
@@ -492,15 +514,16 @@
-  time_zone:
+  time_zone: Asia/Africa/Europe
   track: Beginner
+  keywork: Alidri
```
```suggestion
  keyword: Alidri
```

hectormz commented on _data/speakers.yml:
```diff
@@ -522,8 +545,9 @@
-  time_zone:
+  time_zone: Asia/Africa/Europe
   track: Beginner
+  keywork: Evdaka
```
```suggestion
  keyword: Evdaka
```

hectormz commented on _data/speakers.yml:
```diff
@@ -534,16 +558,17 @@
-  time_zone:
+  time_zone: Asia/Africa/Europe
   track: Advanced
+  keywork: Vindam
```
```suggestion
  keyword: Vindam
```

hectormz commented on _data/speakers.yml:
```diff
@@ -554,15 +579,16 @@
-  time_zone:
+  time_zone: Americas
   track: Beginner
+  keywork: Zheang  
```
```suggestion
  keyword: Zheang  
```

hectormz commented on _data/speakers.yml:
```diff
@@ -573,14 +599,15 @@
-  time_zone:
+  time_zone: Asia/Africa/Europe
   track: Beginner
+  keywork: Thocki
```
```suggestion
  keyword: Thocki
```

hectormz commented on _data/speakers.yml:
```diff
@@ -591,16 +618,17 @@
-  time_zone:
+  time_zone: Asia/Africa/Europe
   track: Beginner
+  keywork: Robkov
```
```suggestion
  keyword: Robkov
```

hectormz commented on _data/speakers.yml:
```diff
@@ -611,15 +639,16 @@
-  time_zone:
+  time_zone: 
   track: Beginner
+  keywork: Eliova
```
```suggestion
  keyword: Eliova
```

OriolAbril commented:
I think the `keywork` is probably a typo on my side, I will merge this and change locally from the text editor which will be faster, thanks @AbhayGoyal and @hectormz. I'll try to rebase and solve the merge issues on your other PR too

hectormz commented:
Got it, sounds good @OriolAbril !
